.travis.yml

```diff
@@ -3,8 +3,7 @@ language: go
 go_import_path: k8s.io/client-go
 
 go:
-  - 1.11.1
+  - 1.11.2
 
 script:
-  - if [ "$TRAVIS_BRANCH" != "master" ]; then godep restore; fi
```

Comment:
Travis has [support for godep](https://docs.travis-ci.com/user/languages/go/#godep-support) already where it installs godep and runs `godep restore` behind the scenes.

If we have this line, it complains with a `godep not found` for non-master branches and requires an explicit install step for non-master branches....and if we have an explicit install step, the automatic godep support for Travis is not triggered.

From https://docs.travis-ci.com/user/languages/go/#godep-support:

> Note that the godep support is only activated if a custom install step is not specified.